### DISCUSSION TITLE
Can't login anymore.

### DISCUSSION THREAD

**Discussion by Snekutaren** [legendary-gl/legendary#470] [Q&A]
I have tried from two different Linux systems and from Windows as well.
This is my debug message:

[sne@snarch ~]$ /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary -v auth
[cli] INFO: Testing existing login data if present...
[LGDLFS] DEBUG: Failed to load user data: FileNotFoundError(2, 'No such file or directory')
[WebViewHelper] DEBUG: Webview unavailable, disabling webview login (Exception: WebViewException('You must have either QT or GTK with Python extensions installed in order to use pywebview.')).
Please login via the epic web login!
If the web page did not open automatically, please manually open the following URL: https://www.epicgames.com/id/login?redirectUrl=https://www.epicgames.com/id/api/redirect
Please enter the "sid" value from the JSON response: Opening in existing browser session.
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
2334a502d69149fb85e4bba289142248
[Core] ERROR: Getting exchange code failed: {'errorCode': 'errors.com.epicgames.unauthorized', 'message': 'You are not authenticated. Please authenticate.', 'correlationId': '19fcfe80-2970-11ed-8775-b311a9114cfb'}
[cli] CRITICAL: No exchange token, cannot login.

**Comment by nutterthanos** [comment]
This issue just got solved 
#468

**Reply by Snekutaren** [reply]
Great! Thanks.
